### DISCUSSION TITLE
FXX East

### DISCUSSION THREAD

**Discussion by Talydaria** [iptv-org/community#862] [Q&A]
### Channel ID

FXXEast.us

### Notes

https://www.fxnetworks.com/fxx

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by SkylerDaKat** [comment]
Damn.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19842
